### ISSUE TITLE
iOS: Activate App event is auto-logged anyway with used FACEBOOK_AUTO_LOG_APP_EVENTS="false"

### ISSUE THREAD

**Issue by tamara-lytvynova** [cordova-plugin-facebook-connect/cordova-plugin-facebook-connect#16]
**Bug or feature request**

[X] I'm reporting a reproducible issue with the code
[ ] I'm reporting a feature request

**Describe the Bug of feature request**
iOS Activate App events are auto-logged with the used `FACEBOOK_AUTO_LOG_APP_EVENTS="false"`. (These events are shown in Facebook Events Manager during testing. Android app works correctly)

The plugin was added with the command below according to the [docs](https://github.com/cordova-plugin-facebook-connect/cordova-plugin-facebook-connect#gdpr-compliance) 

`$ cordova plugin add cordova-plugin-facebook-connect --save --variable APP_ID="***" --variable APP_NAME="***" --variable FACEBOOK_AUTO_LOG_APP_EVENTS="false"`

Looks like `[FBSDKAppEvents activateApp]` method is called in `[FacebookConnectPlugin applicationDidBecomeActive:]` regardless the Facebook setting `isAutoLogAppEventsEnabled`. 

Please compare `applicationDidBecomeActive` method in FacebookConnectPlugin.m and FBSDKApplicationDelegate.m from FBSDKCoreKit.

FacebookConnectPlugin.m
```
- (void) applicationDidBecomeActive:(NSNotification *) notification {
    [FBSDKAppEvents activateApp];
   ...
}
```

FBSDKApplicationDelegate.m from FBSDKCoreKit
```
- (void)applicationDidBecomeActive:(NSNotification *)notification
{
...
  // Auto log basic events in case autoLogAppEventsEnabled is set
  if (FBSDKSettings.isAutoLogAppEventsEnabled) {
    [FBSDKAppEvents activateApp];
  }
...
}
```

**Expected Behavior**

iOS Activate App events shouldn't be auto-logged with the used FACEBOOK_AUTO_LOG_APP_EVENTS="false" variable.

**Plugin version, OS, devices, etc**
cordova-plugin-facebook-connect 1.1.0
cordova version 10.0.0, cordova-ios 6.1.1, iOS 14.3

**Comment by noahcooper** [comment]
Fixed in version 1.1.1.

**Comment by tamara-lytvynova** [comment]
@noahcooper thank you for releasing
